client/src/pages/private/ParticipantDetailsView.js

```diff
@@ -1,19 +1,126 @@
 // Participant Details Page
 // Dependency
-import React /*, { useEffect, useState }*/ from 'react';
+import pick from 'lodash/pick';
+import React, { useEffect, useState } from 'react';
 import { useParams } from 'react-router-dom';
+import { Box, Card, Grid, Link, Typography } from '@material-ui/core';
+import { makeStyles } from '@material-ui/core/styles';
+import store from 'store';
 
 // Libs
-import { Page, CheckPermissions } from '../../components/generic';
+import { Page, CheckPermissions, Alert } from '../../components/generic';
+import { Routes, API_URL } from '../../constants';
+
+// Network call
+const api = async ({ id }) => {
+  const url = `${API_URL}/api/v1/participant?id=${id}`;
+  const resp = await fetch(url, {
+    method: 'GET',
+    headers: {
+      Authorization: `Bearer ${store.get('TOKEN')}`,
+      Accept: 'application/json',
+      'Content-type': 'application/json',
+    },
+  });
+  if (resp.ok) {
+    const [participant] = await resp.json();
+    return participant;
+  } else {
+    throw new Error('Unable to load participant');
+  }
+};
+
+// Key Map
+const keyLabelMap = {
+  fullName: 'Full Name',
+  phoneNumber: 'Phone Number',
+  emailAddress: 'Email Address',
+  interested: 'Program Interest',
+  preferredLocation: 'Preferred Location',
+};
+
+// Display Data
+const displayData = (inputData) => ({
+  ...pick(inputData, Object.keys(keyLabelMap)),
+  fullName: `${inputData.firstName} ${inputData.lastName}`,
+  interested:
+    inputData.interested === 'yes'
+      ? 'Interested'
+      : inputData.interested === 'no'
+      ? 'Not interested'
```

Comment:
Minor thing but in a subsequent PR can we change this display text to "Withdrawn"?

Reply:
Sure